src/Modules/Legacy/Inverse/SolveInverseProblemWithTikhonov.cc

```diff
@@ -124,6 +125,53 @@ void SolveInverseProblemWithTikhonov::execute()
 		sendOutputFromAlgorithm(InverseSolution,output);
 		sendOutputFromAlgorithm(RegularizationParameter,output);
 		sendOutputFromAlgorithm(RegInverse,output);
+    auto lambda=output.get<DenseMatrix>(TikhonovAlgoAbstractBase::RegularizationParameter);
+    auto lambda_array=output.get<DenseMatrix>(TikhonovAlgoAbstractBase::LambdaArray);
+    auto lambda_index =output.get<DenseMatrix>(TikhonovAlgoAbstractBase::Lambda_Index);
+    
+    auto regularization_method  = state->getValue(Parameters::RegularizationMethod).toString();
+
+    if (regularization_method== "lcurve") update_lcurve_gui(lambda,lambda_array,lambda_index);
 
 	}
 }
+
+void SolveInverseProblemWithTikhonov::update_lcurve_gui(const  DenseMatrixHandle& lambda, const DenseMatrixHandle& input, const  DenseMatrixHandle& lambda_index)
```

Comment:
Can these methods be made public/protected to allow unit testing? When we integrate Map3d or whatever graphing module, this code will be the connection point.